packages/client/src/components/designer/menu/default-menu.ts

```diff
@@ -98,6 +99,34 @@ export const DEFAULT_MENU: DesignerMenuItem[] = [
             slots: {} as BlockJSON['slots'],
         },
     },
+    {
+        section: SECTION_INPUT,
+        image: BLOCK_RADIO,
+        name: 'Radio',
+        json: {
+            widget: 'radio',
+            data: {
+                style: {
+                    padding: '4px',
+                },
+                value: '',
+                label: 'Radio Input',
+                isGroup: false,
+                options: [],
```

Comment:
Can we set a default option.

options: [
{label: ''default', value: 'no value'}
]